web/modules/custom/portland/portland.module

```diff
@@ -799,6 +799,13 @@ function portland_form_alter(&$form, &$form_state, $form_id)
     if (isset($form["replace_file"])) {
         $form["replace_file"]['keep_original_filename']["#access"] = false;
     }
+
+    // For exposed search form, force it to search within the group.
```

Comment:
@kxwang , I think you can remove this as all the exposed forms are inline instead of placed as a block now.

Reply:
Code block removed